openbsd.pkr.hcl

```diff
@@ -109,7 +109,13 @@ variable "readonly_boot_media" {
 locals {
   image_architecture = var.architecture == "x86-64" ? "amd64" : var.architecture
   image = "miniroot${replace(var.os_version, ".", "")}.img"
-  image_full_remote_path = "${var.os_version}/${local.image_architecture}/${local.image}"
+
+  image_full_remote_path = (
+    var.is_snapshot == true
```

Comment:
Is this variable necessary? Can we just check if the `checksum` variable is empty or not?

https://developer.hashicorp.com/packer/docs/templates/hcl_templates/functions/collection/length

Reply:
https://github.com/cross-platform-actions/openbsd-builder/pull/19/commits/2979584e5699fc0ce94439a7bde069e2a58ff2b7